buggy_utils.py

```
@@ -0,0 +1,107 @@
+
+
+# XSS vulnerability
+def render_greeting(name):
+    return f"<h1>Welcome, {name}!</h1>"  # unescaped user input
+
+
+# Timing attack vulnerable comparison
+def verify_token(provided, actual):
+    if len(provided) != len(actual):
+        return False
+    for a, b in zip(provided, actual):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)